mkFit/MkFitter.cc

```diff
@@ -65,7 +65,10 @@ void MkFitter::InputTracksAndHits(const std::vector<Track>&  tracks,
   // assert(end - beg == NN);
 
   int itrack;
-#ifdef USE_CUDA
+
+// FIXME: uncomment when track building is ported to GPU.
+#if 0
```

Comment:
For future reference: I'd prefer something like "ifdef USE_CUDA_NOTYET" rather than "if 0"